alpharaw/mzml.py

```diff
@@ -10,10 +10,38 @@
 
 
 class MzMLReader(MSData_Base):
+    """
+    Load mzml file as `:class:`MSData_Base` structure.
+    Register "mzml" in :data:`ms_reader_provider`.
```

Comment:
This registering is strictly speaking not a part of this this class, but of the containing module (and should be refactored, as in alphabase)

Reply:
Yes, we still need to remind users that they can also access this class by using the provider.

Reply:
I will use `register_readers` for all readers in the next PR.